tests/test_infer.py

```diff
@@ -157,3 +157,33 @@ def test_calculator_distributions_without_teststatistic(qtilde):
     )
     with pytest.raises(RuntimeError):
         calc.distributions(1.0)
+
+
+@pytest.mark.parametrize(
+    "backend",
+    [
+        pyhf.tensor.numpy_backend(),
+        pyhf.tensor.tensorflow_backend(),
+        pyhf.tensor.pytorch_backend(),
+        pyhf.tensor.jax_backend(),
+    ],
+)
+@pytest.mark.parametrize(
+    "nsigma,expected_pval",
+    [
+        # values tabulated using ROOT.RooStats.SignificanceToPValue
```

Comment:
ROOT should not be used for calculating anything here. These should be calculated with NumPy/SciPy.

Reply:
my read is that this is for xcheck purposes (i.e. use ROOT to xcheck numpy)

Reply:
yes, that was the intention

Reply:
Why are we using ROOT to validate anything related to just numbers and not implementation though? For something like a p-value I don't think that ROOT should be viewed as Truth.

Reply:
It was just the first one i looked at that didn't seem to have this issue. Anyway, from a quick look, the low-level code use in that function in [scipy](https://github.com/scipy/scipy/blob/0a7bc723d105288f4b728305733ed8cb3c8feeb5/scipy/special/cephes/ndtri.c#L134) and [ROOT](https://root.cern/doc/master/SpecFuncCephesInv_8cxx_source.html#l00137) seems to be basically the same.

Reply:
I don't think it's inherently bad to take the numbers from ROOT. it's not like we're adding a dependency here.

Reply:
It just seems like a strange choice to me given that all our other numbers that we used throughout for testing are taken from NumPy/SciPy and we only use ROOT for validation testing. I doubt there is a difference, but I think we should be consistent and more importantly clear.

Reply:
i added a comment stating that these numbers are consistent with `scipy.stats.norm.sf` and the other test for the "roundtrip" directly uses scipy

Reply:
LGTM